seed/network-connection.sh

```diff
@@ -150,15 +150,18 @@ service_network_netmask=$(CIDR2Netmask $service_network)
 pod_network_address=$(echo $pod_network | cut -f1 -d/)
 pod_network_netmask=$(CIDR2Netmask $pod_network)
 
-node_network_address=$(echo $node_network | cut -f1 -d/)
-node_network_netmask=$(CIDR2Netmask $node_network)
-
 sed -e "s/\${SERVICE_NETWORK_ADDRESS}/${service_network_address}/" \
     -e "s/\${SERVICE_NETWORK_NETMASK}/${service_network_netmask}/" \
     -e "s/\${POD_NETWORK_ADDRESS}/${pod_network_address}/" \
     -e "s/\${POD_NETWORK_NETMASK}/${pod_network_netmask}/" \
-    -e "s/\${NODE_NETWORK_ADDRESS}/${node_network_address}/" \
-    -e "s/\${NODE_NETWORK_NETMASK}/${node_network_netmask}/" openvpn.config.template > openvpn.config
+    openvpn.config.template > openvpn.config
+
+if [[ ! -z "$node_network" ]]; then
```

Comment:
shouldn't we also write it to the openvpn.config in this case? 
```
    -e "s/\${NODE_NETWORK_ADDRESS}/${node_network_address}/" \
    -e "s/\${NODE_NETWORK_NETMASK}/${node_network_netmask}/" openvpn.config.template > openvpn.config
```

Reply:
What do you mean? Can you explain again?

Reply:
you are removing those two lines even if if node_network is set.

Reply:
Yes, see line 163. If `node_network` is set then the pull filter will be added to `openvpn.config`.